src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -3365,6 +3371,14 @@ emit_arm64_intrinsics (
 			ret->sreg3 = scalar->dreg;
 			return ret;
 		}
+        case SN_VectorTableLookup:
+			if (!type_is_simd_vector (fsig->params [0]))
+				return NULL;
+			return emit_simd_ins_for_sig (cfg, klass, OP_XOP_OVR_X_X_X, INTRINS_AARCH64_ADV_SIMD_TBL1, 0, fsig, args);
```

Comment:
I am curious if this change is necessary. If I understand this change correctly, the only thing changed was the value of `inst_c1` field, which was actually ignored by `OP_XOP_OVR_X_X_X` in mini-llvm.c.

Reply:
Discussed offline.